client/trino-cli/src/main/java/io/trino/cli/ClientOptions.java

```diff
@@ -273,6 +273,9 @@ public class ClientOptions
     @Option(names = "--disable-auto-suggestion", description = "Disable auto suggestion")
     public boolean disableAutoSuggestion;
 
+    @Option(names = "--follow-redirects", description = "Follow client redirects from server")
+    public Optional<Boolean> followRedirects;
```

Comment:
Can this be `boolean`?

Reply:
Yeah, with the positive flag naming convention `null/empty` or `true` meant flag is set,  and explicit `false` meant flag is disabled. Hence went with `Boolean` instead of `boolean`

With a negative flag name e.g. `--disable-follow-redirect`, we can use `boolean` instead of `Boolean`